web_src/js/features/common-global.js

```diff
@@ -119,7 +112,11 @@ export function initGlobalCommon() {
   $uiDropdowns.filter('.jump').dropdown({
     action: 'hide',
     onShow() {
-      $('.tooltip').popup('hide');
+      this?._tippy.hide();
+      this?._tippy.disable();
```

Comment:
Why `this?.`? `this._tippy?.`?

Reply:
One more thing, does `this` work in the context? I do not think `this` refers to the correct element .....

Reply:
`this` refers to the button that triggered the dropdown. If the button has no active tippy instance associated `btn._tippy` will be undefined and the `?.` makes it fail gracefully.

It definifely works, the purpose of this is to hide a button tooltip when a dropdown on the same button is opened, like the navbar buttons. Not sure if there is a more elegant way to write this is I see no other way to obtain the button reference. Previous solution was sloppy in that regard that it had hidden all tooltips globally.

Reply:
OK, then `this._tippy?.` should be fine. But the code is using `this?._tippy`.

Reply:
`this._tippy?.` would error out if the `_tippy` property is not present which would be the case for buttons without tooltips. I think the code is fine already, but should probably have a comment on what's happening.

Reply:
IMO the `?.` should be used for the last uncertain property. If `this` is certain, it should not be used with the `?.`

![image](https://user-images.githubusercontent.com/2114189/180783436-9f8cbc23-243b-4ebf-95e0-7a745f5d0ff8.png)

Reply:
Fixed, I added another `?.`, better safe than having a JS error there :)

Reply:
Sorry but I do not think so .....

If `this` is null/undefined, then there must be low-level bug in code. I would like to expose bugs instead of hiding them.

Reply:
done